service/src/test/java/uk/nhs/adaptors/gp2gp/ehr/mapper/EncounterComponentsMapperTest.java

```diff
@@ -405,6 +405,25 @@ public void When_MappingWithRelatedProblemWithIncorrectProblemExtensionUrl_Expec
             .isEqualToIgnoringWhitespace(expectedXml);
     }
 
+    @Test
+    public void When_MappingWithRelatedProblemWithIncorrectProblemExtensionExtensionUrl_Expect_UnspecifiedProblemWithOriginalText() {
```

Comment:
When mapping what? 😄 (also can we make this package-private)

Reply:
When a related problem has an extension (with a correct problem extension url) which itself has an extension which has an incorrect URL!! (hard to fit into a test name with limited characters)
Made the added tests package private (even though it feels so wrong not having a access modifier)

Reply:
I think we've discussed this on a call before, all examples provided by JUnit 5 tend to be package-private! I can see the benefit in them not being public (not totally against it), other packages shouldn't be concerned about the tests!